src/user/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface IPasswordChange {
```

Comment:
```suggestion
export interface IUpdatePasswordDto {
```

''DTO" pour Data Transfer Object. On utilise couramment ce nommage pour faire référence aux payload de requête par exemple.